lib/common/acl.c

```diff
@@ -321,13 +321,21 @@ pcmk__unpack_acl(xmlNode *source, xmlNode *target, const char *user)
                     || !strcmp(tag, XML_ACL_TAG_USERv1)) {
                     const char *id = crm_element_value(child, XML_ATTR_ID);
 
+                    if (id == NULL) {
+                        id = crm_element_value(child, XML_ATTR_NAME);
+                    }
```

Comment:
This should be flipped -- id will (or should) always be there. We should use name if it's present, and id if it's not.